packages/core/stories/card/card.stories.tsx

```diff
@@ -191,72 +227,179 @@ export const InteractableDisabled: StoryFn<typeof InteractableCard> = (
-          onChange={handleChange}
-        />
-        <RadioButton
-          key="left"
-          label="left"
-          value="left"
-          onChange={handleChange}
-        />
-        <RadioButton
-          key="right"
-          label="right"
-          value="right"
-          onChange={handleChange}
-        />
-      </RadioButtonGroup>
+    <StackLayout>
+      <StackLayout gap={1}>
+        <H3>Payment method</H3>
+        <Text>Choose your payment method from the options below.</Text>
+      </StackLayout>
+      <InteractableCardGroup
+        onChange={(_event, value) => {
+          setSelected(value);
+        }}
+      >
+        <InteractableCard {...args} value="card" style={{ width: "180px" }}>
+          <StackLayout gap={1}>
+            <StackLayout gap={1} direction="row" align="center">
+              <CreditCardIcon size={2} />
+              <H3>Credit Card</H3>
+            </StackLayout>
+            <StackLayout direction="row" gap={1}>
+              <RadioButtonIcon checked={selected === "card"} />
+              <Text>Link credit card to your payments account.</Text>
+            </StackLayout>
+          </StackLayout>
+        </InteractableCard>
+        <InteractableCard value="wire" style={{ width: "180px" }}>
+          <StackLayout gap={1}>
+            <StackLayout gap={1} direction="row" align="center">
+              <BankIcon size={2} />
+              <H3>Bank wire</H3>
+            </StackLayout>
+            <StackLayout direction="row" gap={1}>
+              <RadioButtonIcon checked={selected === "wire"} />
+              <Text>Link your bank to your payments account.</Text>
+            </StackLayout>
+          </StackLayout>
+        </InteractableCard>
+        <InteractableCard value="crypto" style={{ width: "180px" }}>
+          <StackLayout gap={1}>
+            <StackLayout gap={1} direction="row" align="center">
+              <DiamondIcon size={2} />
+              <H3>Cryptocurrency</H3>
+            </StackLayout>
+            <StackLayout direction="row" gap={1}>
+              <RadioButtonIcon checked={selected === "crypto"} />
+              <Text>Link your crypto to your payments account.</Text>
+            </StackLayout>
+          </StackLayout>
+        </InteractableCard>
+      </InteractableCardGroup>
     </StackLayout>
   );
 };
 
-export const InteractableAsBlockLink: StoryFn<typeof InteractableCard> = (
+export const InteractableCardGroupCheckbox: StoryFn<typeof InteractableCard> = (
   args
 ) => {
+  const [selected, setSelected] = useState<InteractableCardValue>();
+
   return (
-    <Link style={{ textDecoration: "none" }} href="#" IconComponent={null}>
-      <InteractableCard {...args} style={{ width: "266px" }}>
+    <InteractableCardGroup
+      onChange={(_event, value) => {
+        setSelected(value);
+      }}
+      selectionVariant="multiselect"
+    >
+      <InteractableCard {...args} value="jane-doe" style={{ width: "180px" }}>
```

Comment:
there seems to be different interaction on radio button vs checkbox, may be intended as i know they are different for the controls themselves, but unsure:
- radio button allows for keyboard nav via arrows, but doesn't focus next on tab
- checkbox doesn't allow for nav via arrows, but focuses on next on tab